src/root/rootobject.d

```diff
@@ -12,6 +12,8 @@ module ddmd.root.rootobject;
 
 import core.stdc.stdio;
 
+import ddmd.globals;
```

Comment:
This is wrong, root should not be importing from dmd proper.  DYNCAST should be moved here instead.